docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift cards are available in Saleor 3.1.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiry dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create gift cards?
+Once you have created a product type that can be used as a gift card or voucher, you can issue a gift card. Go to the Gift Card page of the Catalog section. 
+
+![Gift cards](../screenshots/catalog-giftcards-main.png)
+
+Then click the Issue Card button on the top right corner of the page. A pop-up page will appear where you can carry out the following actions related to the gift card creation:  
+
+- Choose currency  
+
+- Enter the amount  
+
+- Send gift cards to customer  
+
+- Set expiry date  
+
+- Write notes
+
+- Assign activation requirement
+
+![Gift cards](../screenshots/catalog-giftcards-new.png)
+
+You can also create multiple gift cards at the same time by clicking the three dots button on the top right corner of the page, near the Issue Card button. Then click the *Bulk Issue* button.
+
+![Gift cards-bulk issue](../screenshots/catalog-giftcards-bulk-issue.png)
+
+Once a gift card has been created you can manage its balance on the top of the gift card details page. You can also add multiple tags (optional) and set an expiration date.  
+
+On the right side of the gift card details page, you can deactivate a gift card, resend a code, and find basic information about the gift card such as:
+
+- Creation date
+
+- Link to order number to which gift card has been used
+
+- Product bought to receive gift card
+
+- User who issued a gift card
+
+- Customer who used a gift card
+
+![Gift cards-details view](../screenshots/catalog-giftcards-details-view.png)
+
+### How to manage metadata in the gift card details page
+Once a gift card has been created you can add private and public metadata for each gift card in the gift card details page. Use the button Add Field to add a new metadata field.
+
+![Gift cards](../screenshots/catalog-giftcards-metadata.png)
+
+### How to activate, deactive or remove gift cards
+Select a single or multiple previously created gift cards from the gift cards list and then click *activate*, *deactivate* or remove by clicking on the trash bin icon.
+
+![Gift cards](../screenshots/catalog-giftcards-deactivate.png)
+
+### How to set an automatic expiration for gift cards
+Click the three dots button on the top right corner of the page, near the Issue Card button. Click *Settings* and then you can set gift cards to expire after a certain time after their purchase. Remember that in some countries, gift cards expiry is prohibited by law.
```

Comment:
Still no mention of the fact that only newly created gift cards will be affected by this change